client/admin.go

```diff
@@ -20,13 +20,17 @@ import (
 	svcv1 "github.com/cerbos/cerbos/api/genpb/cerbos/svc/v1"
 )
 
-const addPolicyBatchSize = 10
+const (
+	addPolicyBatchSize = 10
+	addSchemaBatchSize = 10
+)
 
 type AdminClient interface {
 	AddOrUpdatePolicy(context.Context, *PolicySet) error
 	AuditLogs(ctx context.Context, opts AuditLogOptions) (<-chan *AuditLogEntry, error)
 	ListPolicies(ctx context.Context) ([]string, error)
 	GetPolicy(ctx context.Context, ids ...string) ([]*policyv1.Policy, error)
+	AddOrUpdateSchema(ctx context.Context, schemas []*schemav1.Schema) error
```

Comment:
Because we have the `PolicySet` helper to make it easier to work with `AddOrUpdatePolicy`, I think we should also do something similar for the `AddOrUpdateSchema` call as well to maintain symmetry. It will also hide away the implementation detail of `schemav1.Schema` and let users just deal with files. I think something like:

```go
schemaSet.AddFile("path/to/file.json")
schemaSet.AddReader("id_of_fle", reader)
```